PoESkillTree.Engine.GameModel/PoESkillTree.Engine.GameModel.csproj

```diff
@@ -38,6 +38,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" />
```

Comment:
The System.Numerics namespace in .NET Core doesn't include the Vector2,3,4 types. We are using the Vector2 type for position on JsonPassiveTreePosition class. It is weird because all the other .net runtimes (framework, and xamarin) have Vector2 included in the namespace, but you need a package for .NET Core.